packages/loader/src/requirejs.ts

```diff
@@ -206,7 +206,7 @@ export function amd(altUrl: string = "", min: boolean = true, additionalPaths: {
     const _loaderUrl = altUrl ? altUrl + "/loader" : loaderUrl;
     if (_loaderUrl.indexOf("unpkg.com/@hpcc-js/loader") > 0) {
         return unpkgPath(_loaderUrl + "/package.json", min, additionalPaths);
-    } else if (!altUrl && _loaderUrl.indexOf("file://") === 0) {
+    } else if (!altUrl && (_loaderUrl.indexOf("file://") === 0 || _loaderUrl.indexOf("http://localhost") === 0)) {
```

Comment:
These changes tend to no get highlighted by GitHub very well - it would be better if you suggested the change in the comments as part of the review?